### PR TITLE
[DOC Release] Document `Ember.getWithDefault`

### PR DESCRIPTION
**Pull request by XrXr**
Was strolling around add-on code and found that this isn't documented.

Is this is meant to be public?

### CODE REVIEW

File: packages/ember-metal/lib/property_get.js [hunk 1]
```diff
@@ -159,6 +159,20 @@ export function _getPath(root, path) {
   return root;
 }
 
+/**
+  Retrieves the value of a property from an Object, or a default value in the
+  case that the property returns `undefined`.
+  ```javascript
+  Ember.getWithDefault(person, 'lastName', 'Doe');
+  ```
+  @method getWithDefault
```
Comment (rwjblue) [inline packages/ember-metal/lib/property_get.js]: There needs to be a blank line between fenced code blocks and surrounding text.

File: packages/ember-metal/lib/property_get.js [hunk 2]
```diff
@@ -159,6 +159,20 @@ export function _getPath(root, path) {
   return root;
 }
 
+/**
+  Retrieves the value of a property from an Object, or a default value in the
+  case that the property returns `undefined`.
+  ```javascript
```
Comment (rwjblue) [inline packages/ember-metal/lib/property_get.js]: Please add a blank line above this code fence block.

**Comment by XrXr** [comment]
@rwjblue fixed. Sorry for the churn

**Comment by XrXr** [comment]
Anything blocking this from being merged? @rwjblue

**Comment by rwjblue** [comment]
Nope. :)

**Pull request merged**